data/json/items/tool/lighting.json

```diff
@@ -1081,5 +1081,52 @@
       { "type": "firestarter", "moves": 100 }
     ],
     "flags": [ "LIGHT_15", "TRADER_AVOID", "ALLOWS_REMOTE_USE", "FIRESTARTER", "FIRE" ]
+  },
+  {
+    "id": "hobo_candle",
+    "type": "TOOL",
+    "name": { "str": "hobo candle" },
+    "looks_like": "jar_3l_glass_sealed",
+    "description": "An gasoline based hobo candle, akin to a makeshift alcohol lamp, held together by the hopes, dreams and gum of a survivor.  It doesn't provide very much light, but it can burn for about a week.  You'll need a lighter or matches to light it.",
+    "weight": "965 g",
+    "volume": "3530 ml",
+    "longest_side": "21 cm",
+    "price": "20 USD",
+    "price_postapoc": "80 cent",
+    "material": [ "glass" ],
+    "symbol": "G",
+    "color": "white",
+    "ammo": [ "gasoline" ],
+    "pocket_data": [ { "pocket_type": "MAGAZINE", "watertight": true, "ammo_restriction": { "gasoline": 3000 } } ],
+    "use_action": {
+      "target": "hobo_candle_lit",
+      "msg": "You light the hobo candle.",
+      "active": true,
+      "need_fire": 1,
+      "need_charges": 1,
+      "menu_text": "Light",
+      "type": "transform"
+    },
+    "to_hit": { "grip": "bad", "length": "hand", "surface": "any", "balance": "clumsy" },
+    "melee_damage": { "bash": 4 }
+  },
+  {
+    "id": "hobo_candle_lit",
+    "type": "TOOL",
+    "copy-from": "hobo_candle",
+    "name": { "str": "hobo candle" },
+    "description": "An gasoline based hobo candle, akin to a makeshift alcohol lamp, held together by the hopes, dreams and gum of a survivor.  It doesn't provide very much light, but it can burn for about a week.  This hobo candle is lit.",
+    "turns_per_charge": 238,
+    "use_action": [
+      { "type": "firestarter", "moves": 200 },
+      {
+        "ammo_scale": 0,
+        "target": "hobo_candle",
+        "msg": "The hobo candle winks out.",
+        "menu_text": "Extinguish",
+        "type": "transform"
+      }
+    ],
+    "extend": { "flags": [ "LIGHT_8", "WATER_EXTINGUISH", "TRADER_AVOID", "WIND_EXTINGUISH", "FIRESTARTER" ] }
```

Comment:
> Also, a hood could be formed from the other half of the can.

Shouldn't that make this not extinguishable by the wind 🤔

Reply:
I meant as as extinguishing hood analogous to what alcohol lanterns (or alcohol burners, as I found out they are more commonly called) have for suffocating the flame and keeping the fumes from the drenched string from escaping too quickly. 
Although a proper hood might be also an option by stabbing several holes into the top half of the can. The crafting time might need to reflect the tedious process of creating such a hood though. I also don't know if a gasoline flame, burning directly onto a aluminium cap wouldn't just melt through it eventually.